Web/starter_project_iconics/pages/profile/index.tsx

```diff
@@ -0,0 +1,35 @@
+import BlogCard from '@/components/common/blogCard'
+import { card1 } from '@/store/mockData/data'
+
+const index = () => {
+  const nums: number[] = [0, 1, 2, 3, 4]
+  return (
+    <div
+      style={{
+        display: 'flex',
+        marginTop: '10px',
+        flexDirection: 'row',
+        justifyContent: 'flex-start',
+        flexWrap: 'wrap',
+        alignContent: 'flex-end',
+      }}
+    >
+      {nums?.map((value, index) => {
+        if (index % 2 === 0)
+          return (
+            <div style={{ marginLeft: '7px' }} key={value}>
```

Comment:
[nit] I think it might be a better idea to add the margin to the card itself (and the key as well)
i don't see a reason why we would need a wrapper component for it